src/wp-includes/rest-api/endpoints/class-wp-rest-font-collections-controller.php

```diff
@@ -97,11 +99,21 @@ public function get_items( $request ) {
 			if ( is_wp_error( $item ) ) {
 				continue;
 			}
+
+			/**
```

Comment:
```suggestion
			/*
```

Nit-pick

Reply:
I agree.
I’m accustomed to using DocBlock-style comments because they are correctly parsed by the PHP tokenizer. However, I acknowledge that this is not a true DocBlock comment since it doesn’t contain any PHPDoc tags. Therefore, using a regular multiline comment is more appropriate in this case.  

I’ve created an issue addressing a related problem (posting it here for context and reference): https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/780.